### PR TITLE
[Bug 1201151] Mark html strings in templates as safe.

### PR DESCRIPTION
**Pull request by mythmon**
r?

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ module.exports = function(grunt) {
         },
         watch: {
             scripts: {
-                files: ['js/app.js', 'app/views/*'],
+                files: ['js/app.js', 'app/views/**'],
```
Comment (mythmon) [inline Gruntfile.js]: Without this change, Grunt doesn't re-render the templates in `app/views/main` in the watch mode.

File: app/views/main/question.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
   </main>
   <footer class="QuestionFooter">
     <form id="question_form" class="hbox">
-      <textarea id="question_field" class="QuestionField" rows="1" placeholder="{{ _('Write your message&hellip;') }}"></textarea>
+      <textarea id="question_field" class="QuestionField" rows="1" placeholder="{{ _('Write your message…') }}"></textarea>
```
Comment (mythmon) [inline app/views/main/question.html]: Nunjucks does the `…` to `&hellip;` conversion for us, right?

File: Gruntfile.js (hunk 1 above)
Comment (rehandalal) [inline Gruntfile.js]: d'oh!

File: app/views/main/question.html (hunk 2 above)
Comment (rehandalal) [inline app/views/main/question.html]: yep

**Comment by rehandalal** [comment]
lgtm. r+

**Pull request merged**